server/graphql/types/src/types/item.rs

```diff
@@ -91,6 +92,27 @@ impl ItemNode {
         ))
     }
 
+    pub async fn available_stock_on_hand(
```

Comment:
Remove stock on hand from stats node? 🤔

Reply:
Is it not used anywhere? if not.. go for it!